bin/htcondor_make_runtime.sh

```diff
@@ -10,11 +10,11 @@ STARTDIR=$PWD/tmp/runtime
 CRAB3_VERSION=3.3.0-pre1
 
 WMCOREDIR=$STARTDIR/WMCore
-WMCOREVER=1.1.14.crab1
+WMCOREVER=1.2.6
 WMCOREREPO=dmwm
 
 CRABSERVERDIR=$STARTDIR/CRABServer
-CRABSERVERVER=3.3.1706.rc2
+CRABSERVERVER=3.3.1909.rc1
```

Comment:
I quite confidend that the CRABSERVER version here is only a dummy placeholder. We could probably rewrite the script w/o it, but in any case better have it pointing to the 2 years old one to make it clear.

Reply:
mmm.. don't know.. without this the tarball made by hand was not working for me... And anyway, if it is not used we either keep it updated or we remove it. Am I missing something?

Reply:
maybe we should review your updateRuntime.sh. Yes this part could use some cleanup. So far it was ... it works, don't fix it.